lablqml/src_tests/test3.qml

```diff
@@ -0,0 +1,39 @@
+import QtQuick 2.1
+import QtQuick.Controls 1.0
+
+/* Demonstraction of direct property binding
+OCaml binds to objects named "test" and "mirror".
+When test changes, the binding function is called and the
+mirror's text value is set */
+
+ApplicationWindow {
+    visible: true
+    QtObject {
+	id: t
+	objectName: "test"
+	property string msg: "Hello world! Hello Qml with OCaml!"
+	property QtObject nested: QtObject {
+	    property string nested_msg: "I'm nested"
+	}
+    }
+
+    Column {
+	objectName: "mirror"
+	Text { text: t.msg }
+	Text { objectName: "mirror"; text: "mirror" }
```

Comment:
It's part of the test. The function is supposed to find children with the object name by searching breadth first instead of depth first. Also demonstrates that structure matters.